crates/security_goplus/src/models.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SecurityResponse {
```

Comment:
should this be generic error response across the app?

Reply:
This format is goplus api response, we can define a common one in `primitives` + typeshare for the app

Reply:
yeah let's use this as as standard for api response

Reply:
done, added `SecurityResponse` to `primitives` and api returns same model